### PR TITLE
Adding new variables to the Workflow and saving the arrays

### PR DESCRIPTION
**Pull request by ValVau**

### CODE REVIEW

**Comment by andreypz** [comment]
Thanks for making this MR.
As we discussed in the meeting, I think it would be better to have a low-pt and high-pt categories in a config file. Could you implement this?

In this case you would not need different variables for high and low regions. There will be only one variable name, just created in different sub-categories.
To implement this , you would need to modify the [TwoLepTwoJets](https://github.com/cms-rwth/VHccPoCo/blob/main/CommonSelectors.py#L39-L49) method, introducing the upper pt cut on dilepton (similar to what is done for mll cuts).
Then you would need to define two version of [these cuts](https://github.com/cms-rwth/VHccPoCo/blob/f045f35138b8f2c39e262cf6e1cd4f77a681f4c9/CommonSelectors.py#L233-L255) for lo and high pt regions.

File: cfg_VHcc_ZLL.py [hunk 1]
```diff
@@ -166,5 +191,30 @@
                                    bins=[0,80,150,200,300,450,700],
                                    type="variable",
                                    label="Jets $H_T$ [GeV]")]),
+        "dphi_jj_dr_jj": HistConf([ Axis(field="dijet_deltaR_low", bins=50, start=0, stop=5, label=r"$\Delta R_{jj}$, 50 < $p_{T_{\ell\ell}}$ < 150"),
+                              Axis(field="dijet_deltaPhi_low", bins=64, start=-1, stop=3.5, label=r"$\Delta \phi_{jj}$, 50 < $p_{T_{\ell\ell}}$ < 150")]),
     }
 )
+
+
+run_options = {
```
Comment (andreypz) [inline cfg_VHcc_ZLL.py]: The run options are no longer used in this config. Could you remove this part please.
Instead the `params/run_options.yaml` can be used for additional options (that are different from default in pocket-coffea.)

File: workflow.py [hunk 2]
```diff
@@ -90,3 +198,20 @@ def define_common_variables_after_presel(self, variation):
         #print("CvsL sort CvsL:", self.events["JetsCvsL"][self.events["nJetGood"]>=3].btagDeepFlavCvL)
 
         self.events["dijet_csort"] = get_dijet(self.events.JetsCvsL)
+        
+        # Suppress FutureWarning
```
Comment (andreypz) [inline workflow.py:222]: We should have an option in the config file to create the `root` or not. Could you introduce that as well?

**Review by andreypz — CHANGES_REQUESTED** [review]
I have added a couple more comments on this request. Please take a look @ValVau .
Also, there are conflicts of the changes wrt main branch. Could you please rebase/resolve the conflicts.

**Comment by ValVau** [comment]
Take a look at the new commit. There I have done as you requested: new function in the CommonSelectors, new config and changed workflow.

**Comment by andreypz** [comment]
Thanks, this is probably fine for now.
Could you please resolve the conflicts and then we can merge it

**Comment by ValVau** [comment]
In the second commit, CommonSelectors and cfg_ZLL include both your and my changes together, so no conflicts should arise.

**Pull request merged**